ctpolicy/ctpolicy_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package ctpolicy
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/google/certificate-transparency-go/loglist"
+	"github.com/google/certificate-transparency-go/testdata"
+	"github.com/google/certificate-transparency-go/x509"
+	"github.com/google/certificate-transparency-go/x509util"
+)
+
+func getTestCertPEMShort() *x509.Certificate {
+	cert, _ := x509util.CertificateFromPEM([]byte(testdata.TestCertPEM))
```

Comment:
BTW, an alternative way of dealing with the error is to make the method a helper:
```golang
func getTestCertPEMShort(t *testing.T) *x509.Certificate {
	t.Helper()
	cert, err := x509util.CertificateFromPEM([]byte(testdata.TestCertPEM))
	if err != nil {
		t.Fatalf("Failed to parse hardcoded test cert:%v", err)
	}
	cert.NotAfter = time.Date(2013, 1, 1, 0, 0, 0, 0, time.UTC)
	return cert
}
```

(but leaving as-is is fine with me too)

Reply:
Thanks for the note.